docs/docs/documentation/getting-started/installation/postgres.md

```diff
@@ -5,40 +5,41 @@ PostgreSQL might be considered if you need to support many concurrent users. In
 **For Environment Variable Configuration, see** [Backend Configuration](./backend-config.md)
 
 ```yaml
----
-version: "3.7"
 services:
   mealie:
     image: ghcr.io/mealie-recipes/mealie:v1.3.2 # (3)
     container_name: mealie
+    restart: always
     ports:
         - "9925:9000" # (1)
     deploy:
       resources:
         limits:
           memory: 1000M # (2)
-    depends_on:
-      - postgres
     volumes:
       - mealie-data:/app/data/
+      - /etc/timezone:/etc/timezone:ro
     environment:
-    # Set Backend ENV Variables Here
-      - ALLOW_SIGNUP=true
-      - PUID=1000
-      - PGID=1000
-      - TZ=America/Anchorage
-      - MAX_WORKERS=1
-      - WEB_CONCURRENCY=1
-      - BASE_URL=https://mealie.yourdomain.com
+      # Set Backend ENV Variables Here
+      ALLOW_SIGNUP: true
+      PUID: 1000
+      PGID: 1000
+      MAX_WORKERS: 1
+      WEB_CONCURRENCY: 1
+      BASE_URL: http://localhost # (4)
+      # Database Settings
+      DB_ENGINE: postgres
+      POSTGRES_URL: 'postgres://mealie:mealie@postgres/mealie' # (5)
```

Comment:
If default is none, we probably shouldn't be setting one in the sample file I'd think

Reply:
Because of how the refactor was done, it ends up being that value anyway. The old values were unchanged, it's just now the function returns self.DATABASE_URL always. It's initially none so the function only needs to build it once, and from then on can just reference it for the lifecycle of the PostgresProvider instance. The idea of listing the new format in the example is that it works for more use cases - see the linked issue for one such example.